frontend/src/components/pages/TournamentRegistrations/TournamentRegistrations.tsx

```diff
@@ -77,6 +78,11 @@ export const TournamentRegistrations = () => {
     }
   }
 
+  const resetInput = () => {
+    //TODO dochodzę tutaj i oficjalnie czyszczę input, ale problem jest taki, że nie mogę w nim nic pisać (input jest cały czas pusty), bo cały czas na nowo się renderuje ten kompoennt
```

Comment:
@nowakprojects wg mnie taki sposób jest przekombinowany, ale nie wiem jak inaczej się dostać w drzewie do dziadka, żeby mu input wyczyścić to raz, a dwa jak póki co i tak to nie działa, bo przez to, że mam na inpucie value={searchInputValue} to przez to, że cały czas podczas pisania w inpucie komponent się renderuje input jest pusty...
Będę wdzięczny za dalsze wskazówki :-)

Reply:
A kuzwa... nie widziałem, ze to jest aż "dziadek". No ale niestety raczej inaczej się na teraz nie da. Tzn. z takim rozwiązaniem. Dochodzimy trochę do takiego callback hell.
Roziwązaniem jest:
- Redux
- Mniej inwazyjny jest Hook: useReducer i useContext:
     - https://typeofweb.com/usereducer-przenoszenie-logiki-poza-komponent/
     - https://reactjs.org/docs/hooks-reference.html#usereducer

Reply:
Super - dzięki. Popatrzę, poczytam, pozmieniam :-)